.github/pull_request_template.md

```diff
@@ -17,3 +17,4 @@ Tell us what this PR does or link to any related issues that describe the goal h
 - [ ] Added any changes in this PR to the `CHANGELOG.md` `Next` section
 - [ ] If this pull request includes a new component or feature, has it been exported from one of the library's entry points?
 - [ ] Does the component dispatch relevant interaction events? (ie: on:click, on:change, etc.)
+- [ ] Does the component directory include a separate `.docs.md` imported into `.stories.svelte` for description and usage information?
```

Comment:
Suggested tweak: Make this language more specific about including "description and usage information" but maybe leave out the import seperate `.docs.md` ? I'm thinking about cases where a component needs custom mdx docs for more robust formats (See maps/SVGMap/SVGMap.docs.mdx) where this might not apply.

Reply:
Just updated! Let me know what you think.